rpm/archivematica/etc/archivematica-storage-service.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Archivematica Storage Service
+After=network.target
+
+[Service]
+
+PIDFile=/run/archivematica-storage-service_gunicorn.pid
+User=archivematica
+Group=archivematica
+EnvironmentFile=-/etc/sysconfig/archivematica-storage-service
+WorkingDirectory=/usr/share/archivematica/storage-service/
+ExecStart=/usr/lib/python2.7/archivematica/storage-service/bin/gunicorn --workers 2 --timeout 120 --access-logfile /var/log/archivematica/storage-service/gunicorn.log --error-logfile /var/log/archivematica/storage-service/gunicorn_error.log --log-level error --bind localhost:7500 storage_service.wsgi:application
```

Comment:
Should we use `/etc/archivematica/storage-service.gunicorn-config.py`?

Reply:
Done